test/Scripts.Tests/test-pack-contents.ps1

```diff
@@ -26,7 +26,11 @@ Can't compare package contents against baseline."
 $zip = [IO.Compression.ZipFile]::OpenRead($packageFile)
 try {
     $snapshotContent = $zip.Entries.FullName.Replace("\", "/")
+    # Override the generated GUID as part of the .bcsymbolmap
+    $snapshotContent = $snapshotContent -replace "BCSymbolMaps\/.+?(\.)","BCSymbolMaps/*." 
```

Comment:
Filtering here, since Bitcode is now part of the `.xcframework` and the Guid changes with every build.